setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = kpsr_codegen
+version = 1.0
+
+[options]
+install_requires = 
+        Jinja2
+        PyYaml
+
+packages = find:
+author = Klepsydra
+author_email = support@klepsydra.com
+description = Klepsydra SDK Code Generator
+license = Apache License
+python_requires= >=3.6
+include_package_data = True
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[options.entry_points]
+console_scripts = 
+    kpsr_codegen = kpsr_codegen.__main__:main
+
+[options.packages.find]
+exclude = tests*
+
+[options.package_data]
+kpsr_codegen =
+        conf/*
+        templates/*
+        cmake/*
```

Comment:
This adds the cmake/KpsrCodeGen.cmake to the package. But maybe that file should be in kpsr-core ? I'm not sure how to have this .cmake file installed in the correct location for cpp projects/cmake to find.

Reply:
We can create a post install command in the setup.py:
https://stackoverflow.com/questions/20288711/post-install-script-with-python-setuptools
I'll add a JIRA for this.